controllers/ajax.py

```diff
@@ -911,11 +911,11 @@ def getAssessResults():
         res = {'answer': rows[0][0], 'timestamp': str(rows[0][1]), 'correct': rows[0][2]}
         return json.dumps(res)
     elif event == "timedExam":
-        query = "select correct, incorrect, skipped, time_taken, timestamp from timed_exam where div_id=%s and course_name=%s and sid=%s order by timestamp desc"
+        query = "SELECT correct, incorrect, skipped, time_taken, timestamp, reset FROM timed_exam WHERE div_id=%s AND course_name=%s AND sid=%s ORDER BY timestamp DESC"
```

Comment:
All of the queries in this block of `if/elif` statements use the `db.executesql()` function. Would you like me to change them all to the web2py query style?